app/controllers/api/v2/allocations_controller.rb

```diff
@@ -0,0 +1,33 @@
+module API
+  module V2
+    class AllocationsController < API::V2::ApplicationController
+      def create
+        authorize @allocation = Allocation.new(allocation_params.merge(accredited_body_id: accredited_body.id))
```

Comment:
i think we should get this in for now, but a discussion on whether or not this end point should be idempotent may come into play when we look at when users change their request

Reply:
I was thinking to just add a uniqueness validation to prevent duplicates being created once we settle on what a 'duplicate' is with the recruitment cycle stuff and then just add an explicit update action when we're doing the updates. Happy to discuss that along the way though.